tasks/34_max-triplet-sum/max-triplet-sum.js

```diff
@@ -0,0 +1,12 @@
+export function maxTripletSum(arr, n = 3) {
+  const set = arr.filter((el, index) => arr.indexOf(el) === index);
```

Comment:
Will it work all the time? I mean `indexOf`

Reply:
i need to use lastIndexOf, to avoid duplicates